### PR TITLE
SDK-25

### PR DESCRIPTION
**Pull request by sidibos**
Remove Yoti plugin config data when it's deleted from Wordpress

### CODE REVIEW

File: yoti-connect/readme.txt [hunk 1]
```diff
@@ -74,10 +79,21 @@ For FAQ please click [here.](https://yoti.zendesk.com/hc/en-us/categories/201129
 4. Add Yoti button for faster login experience
 
 == Changelog ==
-= 1.1.1 =
-This is the latest version
 
-In this release we removed php module mcrypt dependency from wordpress plugin and fixed a bug that was occurring when a user decides not to link their account to Yoti in the login process.
+Here you found the changes in each version.
```
Comment (zarembas) [inline yoti-connect/readme.txt:83]: Here you can find the changes for each version.

File: yoti-connect/readme.txt [hunk 2]
```diff
@@ -74,10 +79,21 @@ For FAQ please click [here.](https://yoti.zendesk.com/hc/en-us/categories/201129
...
+
+    Version     Date            Changes
+
+    1.1.2       2017/01/08      Remove Yoti plugin config data when the plugin is uninstalled and removed from wordpress.
```
Comment (zarembas) [inline yoti-connect/readme.txt:87]: uppercase Wordpress

File: yoti-connect/readme.txt [hunk 3]
```diff
@@ -74,10 +79,21 @@ For FAQ please click [here.](https://yoti.zendesk.com/hc/en-us/categories/201129
...
+
+    1.1.1       2017/20/07      Fix a bug that was occurring when a user decides not to link their account to Yoti During the login process.
```
Comment (zarembas) [inline yoti-connect/readme.txt:89]: lowercase during

File: yoti-connect/readme.txt [hunk 4]
```diff
@@ -74,10 +79,21 @@ For FAQ please click [here.](https://yoti.zendesk.com/hc/en-us/categories/201129
...
+
+    1.1.0       2017/20/07      Remove php module mcrypt dependency from wordpress plugin
```
Comment (zarembas) [inline yoti-connect/readme.txt:91]: PHP and Wordpress

**Comment by zarembas** [comment]
@sidibos Just a few typos. Maybe you can sort them out in the next release.

**Comment by sidibos** [comment]
@vassyz Ok I will apply those changes - thx and enjoy your holiday :)

**Pull request merged**